index.html

```diff
@@ -180,6 +180,22 @@ <h3>Skills</h3>
             </ul>
         </section>
     </section>
+    <section class="last">
```

Comment:
- [ ] [OPTIONAL] This section has a fixed height of `800px`, and therefore the purple background is being over before the form is over: 

![image](https://user-images.githubusercontent.com/77038610/150627863-2e7355e8-77aa-434b-be21-fea82d31110a.png)

I strongly suggest to delete the ` height: 800px;` property to fix this issue.